# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../log4cxx/gcc44.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fix to build using gcc >= 4.4

Note: Depends on gcc43.patch!

diff -ruN apache-log4cxx-0.10.0-gcc43/src/examples/cpp/console.cpp apache-log4cxx-0.10.0/src/examples/cpp/console.cpp
--- apache-log4cxx-0.10.0-gcc43/src/examples/cpp/console.cpp	2010-08-16 13:35:44.908007262 +0200
+++ apache-log4cxx-0.10.0/src/examples/cpp/console.cpp	2010-08-16 13:39:45.260336125 +0200
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+#include <cstdio>
 #include <cstdlib>
 #include <cstring>
 #include <log4cxx/logger.h>
